actions/builder/update/entrypoint/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/paketo-buildpacks/github-config/actions/builder/update/entrypoint/internal"
+)
+
+type Options struct {
+	BuilderFile    string
+	RegistryServer string
+}
+
+func main() {
+	var options Options
+
+	flag.StringVar(&options.BuilderFile, "builder-file", "", "Path to the builder.toml file")
+	flag.StringVar(&options.RegistryServer, "registry-server", "", "Registry server uri (e.g. gcr.io")
+	flag.Parse()
+
+	if options.BuilderFile == "" {
+		fail(errors.New(`missing required input "builder-file"`))
+	}
+
+	if options.RegistryServer == "" {
+		fail(errors.New(`missing required input "registry-server"`))
+	}
+
+	builder, err := internal.ParseBuilderFile(options.BuilderFile)
```

Comment:
Will this work the first time it runs when there is no builder.toml file yet?

Reply:
The assumption is that the `builder.toml` is checked in from the very beginning. The first time, a human will have created a `builder.toml` by hand.

Reply:
Sounds good, I didn't realize that was the assumption